### PR TITLE
updated docs of automated meshing

### PR DESCRIPTION
**Pull request by maciej-flexcompute**

### CODE REVIEW

File: docs/source/preprocessing/volumeMesherSlidingInterfaces.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+name@ string@ index (0, 1, 2...)@ “interface“@ Name for the sliding interface. The boundary name will become: “slidingInterface-<name>“.
+innerRadius@ float@ REQUIRED@ 0@ Inner radius of sliding interface, if greater than 0 then “donut” shape is created.
+outerRadius@ float@ REQUIRED@ 10@ Outer radius of sliding interface.
+thickness@ float@ REQUIRED@ 0.5@ Thickness of sliding interface.
+axisOfRotation@ 3-array@ REQUIRED@ [1, 0, 0]@ Axis of rotation of the sliding interface. The cylindrical shape is positioned such its axis is aligned with axisOfRotation.
+center@ 3-array@ REQUIRED@ [0, 0, 0]@ Position of center of the sliding interface.
+spacingAxial@ float@ REQUIRED@ 0.01@ Spacing along the axial direction.
+spacingRadial@ float@ REQUIRED@ 0.03@ Spacing along the radial direction.
+spacingCircumferential@ float@ REQUIRED@ 0.03@ Spacing along the circumferential direction.
+enclosedObjects@ []@ REQUIRED@ [“hub“, “blade1“, “blade2“, “blade3“]@ Objects enclosed by this sliding interface. Can be faces, other sliding intefaces or rotor disks.
```
Comment (rundaji) [inline docs/source/preprocessing/volumeMesherSlidingInterfaces.csv:11]: Just a minor issue, but it seems “ is different with "

File: docs/source/preprocessing/volumeMesherRotorDisks.csv [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Option@ Type@ Default@ Example@ Description
+name@ string@ index (0, 1, 2...) @ “rotor“@ Name for the rotor disk. The boundary name will become: rotorDisk-<name>.
```
Comment (rundaji) [inline docs/source/preprocessing/volumeMesherRotorDisks.csv:2]: Similar “ versus " issue

File: docs/source/preprocessing/volumeMesherSlidingInterfaces.csv (hunk 1 above)
Comment (rundaji) [reply docs/source/preprocessing/volumeMesherSlidingInterfaces.csv:11]: Shall we add a screenshot to the "slidingInterfaces" example json file?

File: docs/source/preprocessing/surfaceMesherEdges.csv [hunk 3]
```diff
@@ -1,2 +1,5 @@
 Option@ Type@ Default@ Example@ Description
-<edgeName>-> firstLayerThickness@ float@ REQUIRED@ 1e-3@ First layer thickness of the anisotropic surface cells growing along the orthogonal direction to the edge named <edgeName>. See :numref:`fig1_surfMesh_edges`.
\ No newline at end of file
+<edgeName>-> type@ ["aniso", "projectAnisoSpacing"]@ REQUIRED@ "aniso"@ type="aniso": anisotropic surface cells growing along the orthogonal direction to the edge named <edgeName>. See :numref:`fig1_surfMesh_edges`. type="projectAnisoSpacing": project spacing.
```
Comment (rundaji) [inline docs/source/preprocessing/surfaceMesherEdges.csv:2]: Shall we add a screenshot for "projectAnisoSpacing"?

File: docs/source/preprocessing/automatedMeshing.rst [hunk 4]
```diff
@@ -154,11 +154,18 @@ Example\:
...
+            "value": 1
         },
         "trailingEdge":  {
-            "firstLayerThickness": 1e-3
-        }
+            "type": "aniso",
+            "method": "height",
+            "value": 1e-3
+        },
+        "hubEdge": {
```
Comment (rundaji) [inline docs/source/preprocessing/automatedMeshing.rst:166]: Would be nice if we can have a screenshot for this

File: docs/source/preprocessing/automatedMeshing.rst [hunk 5]
```diff
@@ -268,4 +274,78 @@ Example\:
...
+   :file: ./volumeMesherSlidingInterfaces.csv
+   :header-rows: 1
+   :delim: @
+   :widths: 20 10 10 10 50   
+
+Example\:
+
+.. code-block:: json 
+
+    "slidingInterfaces": [
```
Comment (rundaji) [inline docs/source/preprocessing/automatedMeshing.rst:296]: Is this example a little bit too complicated?

Comment (maciej-flexcompute) [reply docs/source/preprocessing/automatedMeshing.rst:296]: Yep, it is quite complicated. But this example shows full potential and flexibility of rotating interface. I think it if good

File: docs/source/preprocessing/volumeMesherSlidingInterfaces.csv (hunk 1 above)
Comment (maciej-flexcompute) [reply docs/source/preprocessing/volumeMesherSlidingInterfaces.csv:11]: added screenshot

Comment (rundaji) [reply docs/source/preprocessing/volumeMesherSlidingInterfaces.csv:11]: Sorry, where can I find the screenshot?

Comment (maciej-flexcompute) [reply docs/source/preprocessing/volumeMesherSlidingInterfaces.csv]: Sorry, haven't pushed my changes. Now you can see them

**Review by rundaji — APPROVED** [review]

**Pull request merged**